src/NLog/Layouts/SimpleLayout.cs

```diff
@@ -240,6 +240,18 @@ internal void SetRenderers(LayoutRenderer[] renderers, string text)
             }
         }
 
+        /// <inheritdoc />
+        public override object GetRawValue(LogEventInfo logEvent)
+        {
+
+            if (Renderers.Count == 1 && Renderers[0] is IRawValue rawValueLayoutRenderer)
+            {
+                return rawValueLayoutRenderer.GetRawValue(logEvent);
+            }
+
+            return null;
```

Comment:
doubting if this is good enough..

If we get `null`, then we could call render again and that's not always the best solution

Reply:
I think it is a good idea that `Layout.GetRawValue` always return null (Good default for CsvLayout + JsonLayout + etc.). Then SimpleLayout need to do the conversion if ``IRawValue`-optimization is not available.

Reply:
Prefer that the fallback render logic is inside this method, instead of the caller having to handle `null`.

Reply:
I changed it already to TryGetRawValue, for performance reasons.

If it returns null (out) and true, there is no need to use the string render anymore

Reply:
Prefer the old logic, where it tries all kinds of ways, and then fallback to a provided default value.

Reply:
but then I could call twice .render, while that's not needed

Reply:
You are right `TryGetRawValue` will just be a single helper method, that is used by an generic-extension method that gets the default-value as input paramater and can perform rendering if needed.